core/src/main/scala/chisel3/Width.scala

```diff
@@ -20,19 +20,19 @@ sealed abstract class Width {
   def shiftRight(that:         Int): Width = this.op(this, (a, b) => 0.max(a - that))
   def unsignedShiftRight(that: Int): Width = this.op(this, (a, b) => 0.max(a - that))
   def signedShiftRight(that:   Int): Width = this.op(this, (a, b) => 1.max(a - that))
-  def dynamicShiftLeft(that:   Width): Width =
+  def dynamicShiftLeft(that: Width): Width =
     this.op(that, (a, b) => a + (1 << b) - 1)
 
-  def known: Boolean
-  def get:   W
+  def known:                                     Boolean
+  def get:                                       W
   protected def op(that: Width, f: (W, W) => W): Width
 }
 
 case object UnknownWidth extends Width {
-  def known: Boolean = false
-  def get:   Int = None.get
+  def known:                           Boolean = false
+  def get:                             Int = None.get
```

Comment:
It's a bit strange to me to align the types, but not the `=` ? But I suppose that might be a much larger diff, which is fine to avoid.